app/services/CapiHelper.scala

```diff
@@ -0,0 +1,60 @@
+package services
+
+import java.util.concurrent.TimeUnit
+
+import com.gu.contentapi.client.model.v1.{Content, SearchResponse}
+import logging.Logging
+
+import scala.concurrent.duration.Duration
+import scala.concurrent.{Await, ExecutionContext, Future}
+
+object CapiHelper extends Logging {
```

Comment:
Why can't all of this just live in the Capi class?  It's Capi-specific functionality

Reply:
to not make CAPI class to large
It is easier to read while having small objects

Reply:
that's changed according to our discussion